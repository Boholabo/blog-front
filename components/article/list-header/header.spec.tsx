import React from 'react';
import { render } from '@/test/testUtils';
import ListHeader from './index';

describe('article/list-header component', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<ListHeader />, {});
    expect(asFragment()).toMatchSnapshot();
  });
});
