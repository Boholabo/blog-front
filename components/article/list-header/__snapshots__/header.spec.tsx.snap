// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`article/list-header component matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="list-header"
  >
    <div
      class="logo-box"
    >
      <p
        class="logo"
      >
        logo
      </p>
    </div>
  </div>
</DocumentFragment>
`;
